debian/po/pt_BR.po

```diff
@@ -824,22 +823,22 @@ msgid ""
 msgstr ""
 
 #: ../../uaclient/messages/__init__.py:386
-#, fuzzy, python-brace-format
+#, python-brace-format
 msgid ""
 "Failed to automatically attach to an Ubuntu Pro subscription {num_attempts} "
 "time(s).\n"
 "The failure was due to: {reason}.\n"
 "The next attempt is scheduled for {next_run_datestring}.\n"
 "You can try manually with `sudo pro auto-attach`."
 msgstr ""
-"{num_attempts} falha(s) ao vincular automaticamente aos serviços do Ubuntu "
-"Pro.\n"
+"{num_attempts} falha(s) ao tentar vincular automaticamente a uma assinatura "
```

Comment:
Do we really need `tentar` here ?

Reply:
I thought it reads better, we tried and failed :D
Can roll back to the old phrase though

Reply:
Done